client/pages/individual-result.jsx

```diff
@@ -0,0 +1,91 @@
+      })
+      .catch(err => console.error(err));
+  }
+
+  render() {
+    if (!this.state.product) return null;
+    const result = this.state.product._embedded.events[0];
+    let newTimeString = '';
+    const newTime = [];
+    const newDate = [];
+    if (result.dates.start.localTime) {
+      const timeArray = result.dates.start.localTime.split(':');
+      if (timeArray[0] > 12) {
+        newTime.push(timeArray[0] - 12);
+        newTime.push(`${timeArray[1]}pm`);
+      } else {
+        newTime.push(timeArray[0]);
+        newTime.push(`${timeArray[1]}am`);
+      }
+      newTimeString = newTime.join(':') + ' |';
+    }
+    if (result.dates.start.localDate) {
+      const dateArray = result.dates.start.localDate.split('-');
+      newDate.push(`${dateArray[1]}/${dateArray[2]}/${dateArray[0]}`);
+    }
+    if (result.seatmap) {
+      return (
+        <div className='activity-results-container'>
+          <div style={styles.back} className='back'>
+            <a style={styles.anchor} href={`#results?data=${result._embedded.venues[0].city.name},${result._embedded.venues[0].state.stateCode}`}>back</a>
+          </div>
+          <div className='result'>
+            <img className='image-top' src={result.images[0].url} alt="image" />
+            <img className='image-bottom' src={result.seatmap.staticUrl} alt='seatmap' />
+          </div>
+          <div className='information'>
+            <h2 className='name'>{result.name}</h2>
+            <h3 className='venue'>{result._embedded.venues[0].name}</h3>
+            <h3 className='address'>{result._embedded.venues[0].address.line1}, {result._embedded.venues[0].city.name}, {result._embedded.venues[0].state.stateCode}</h3>
+            <h3 className='date-time'>{newTimeString} {newDate[0]}</h3>
+            <a href={result.url}>
+              <button className='get-ticket'>Get A Ticket</button>
+            </a>
+          </div>
+        </div>
+      );
+    } else {
+      return (
+        <div className='activity-results-container'>
+          <div style={styles.back} className='back'>
+            <a style={styles.anchor} href={`#results?data=${result._embedded.venues[0].city.name},${result._embedded.venues[0].state.stateCode}`}>back</a>
+          </div>
+          <div className='result'>
+            <img className='image-solo' src={result.images[0].url} alt="image" />
+          </div>
+          <div className='information'>
+            <h2 className='name'>{result.name}</h2>
+            <h3 className='venue'>{result._embedded.venues[0].name}</h3>
+            <h3 className='address'>{result._embedded.venues[0].address.line1}, {result._embedded.venues[0].city.name}, {result._embedded.venues[0].state.stateCode}</h3>
+            <h3 className='date-time'>{newTimeString} {newDate[0]}</h3>
+            <a href={result.url}>
+              <button className='get-ticket'>Get A Ticket</button>
+            </a>
+          </div>
+        </div>
+      );
+    }
```

Comment:
If the only different is whether or not to include a seat map image, you can use `&&` conditional rendering to include the image.

```jsx
{ result.seatmap && <img className='event-image' src={result.seatmap.staticUrl} alt='seatmap' /> }
```

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

You can use `nth-child` in your CSS to give all but the last image the bottom margin you need.

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child